classes/02class/exercises/c02-network02/drii-cavalcanti/c02-network02.txt

```diff
@@ -0,0 +1,12 @@
+
+IP address          Subnet Address                 How many usable hosts
+10.0.0.0/16         10.0.0.0/24;                            251
```

Comment:
why do you have two different CIDR's here? could you explain? Also, try to add labels for the subnet information on this file so we can easily identify

Reply:
Is your question about VPC's CIDR and Subnets' CIDR?